HW1/P2/P2b.py

```diff
@@ -1,3 +1,39 @@
+# Author: George Lok
+# P2b.py
+
 from P2 import *
 
 # Your code here
+import findspark
+findspark.init('/Users/georgelok/spark')
+
+import pyspark
+sc = pyspark.SparkContext(appName="myAppName")
+
+def cartesianToPolarAngle(x,y,factor) :
+  ratio = x / (y * 1.0)
+  angle = np.arctan(ratio)
+  if(x < 0) :
+    angle += np.pi
+  return int(angle / (2 * np.pi) * 100)%100
+
+# Set 100 partitions
+coordinates = sc.parallelize([(x,y) for x in xrange(2000) for y in xrange(2000)])
+coordinates = coordinates.partitionBy(100)
+results = coordinates.map(lambda (x,y) : ((x,y), mandelbrot((x/500.0) - 2., (y/500.0)-2.)))
+
+# Confirm result is correct
+draw_image(results)
+
+computeData = sum_values_for_partitions(results)
+
+data = computeData.collect()
+effort = np.array([d for d in data])
+
```

Comment:
I really like your histogram. it is a bit different from the one made by the default code provided to us in the homework, but i feel as though your new plot really gives a sense of how evenly distributed the load was across the 100 partitions. nice!